diff -up crda-1.1.1_2009.11.25/crda-1.1.1/udev/regulatory.rules.orig crda-1.1.1_2009.11.25/crda-1.1.1/udev/regulatory.rules
--- crda-1.1.1_2009.11.25/crda-1.1.1/udev/regulatory.rules.orig	2010-01-26 13:01:51.000000000 -0500
+++ crda-1.1.1_2009.11.25/crda-1.1.1/udev/regulatory.rules	2010-01-26 13:01:41.000000000 -0500
@@ -2,4 +2,6 @@
 # For more information see:
 # http://wireless.kernel.org/en/developers/Regulatory/CRDA
 
+SUBSYSTEM=="ieee80211", ACTION=="add", RUN+="/sbin/setregdomain"
+
 KERNEL=="regulatory*", ACTION=="change", SUBSYSTEM=="platform", RUN+="$(SBINDIR)crda"
